configs/containers/docker-ubuntu-gcc-openmpi.yaml

```diff
@@ -47,6 +47,11 @@ spack:
       externals:
       - spec: git-lfs@2.9.2
         prefix: /usr
+    mysql:
+      buildable: false
+      externals:
+      - spec: mysql@8.0.32
```

Comment:
wanted to verify mysql version here, just because we used 8.0.31 for the on-prem machines.

Reply:
Thanks for checking. As long as it's something new (I believe anything above 8 does), we are good. There are minor updates between those versions, but no functional difference. The version that comes with `apt` is `8.0.32`, the one that comes from the mysql community server install is `8.0.31`.